3.3/test/run

```diff
@@ -84,7 +89,8 @@ wait_for_cid() {
   local sleep_time=1
   local attempt=1
   local result=1
-  info "Waiting for application container to start"
+  local cargs=${CONTAINER_ARGS:-}
+  info "Waiting for application container to start $cargs"
```

Comment:
Hmm... I'm wondering why do we need this `local cargs`...

If we had somewhere `set -u`, trying to expand `$CONTAINER_ARGS` when it is not set would be an error. But since we don't, what would be the difference if we simply had `info "Waiting for application container to start $CONTAINER_ARGS"`?

@csrwng, what's the reason behind this? Maybe I can learn something :smile: